src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/request/StrikeOffObjectionsEntity.java

```diff
@@ -0,0 +1,112 @@
+package uk.gov.companieshouse.api.strikeoffobjections.model.request;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.data.mongodb.core.mapping.Field;
+
+import java.time.LocalDateTime;
+
+@Document(collection = "strike_off_objections")
+public class StrikeOffObjectionsEntity {
+
+
+    public static class Builder {
+        private String eMailAddress;
+        private String reason;
+        private CreatedBy createdBy;
+        private LocalDateTime createdOn;
+        private RequestStatus status;
+
+        public Builder withEMailAddress(String eMail) {
+            this.eMailAddress = eMail;
+            return this;
+        }
+
+        public Builder withReason(String requestInformation) {
+            this.reason = requestInformation;
+            return this;
+        }
+
+        public Builder withCreatedBy(CreatedBy createdBy) {
+            this.createdBy = createdBy;
+            return this;
+        }
+
+        public Builder withCreatedOn(LocalDateTime createdOn) {
+            this.createdOn = createdOn;
+            return this;
+        }
+
+        public Builder withStatus(RequestStatus status) {
+            this.status = status;
+            return this;
+        }
+
+        public StrikeOffObjectionsEntity build() {
+            StrikeOffObjectionsEntity strikeOffObjectionsEntity = new StrikeOffObjectionsEntity();
+            strikeOffObjectionsEntity.setEMailAddress(this.eMailAddress);
+            strikeOffObjectionsEntity.setReason(this.reason);
+            strikeOffObjectionsEntity.setCreatedBy(this.createdBy);
+            strikeOffObjectionsEntity.setCreatedOn(this.createdOn);
+            strikeOffObjectionsEntity.setStatus(this.status);
+            return strikeOffObjectionsEntity;
+        }
+    }
+
+    @Id
+    private String id;
+    @Field("e_mail_address")
```

Comment:
email is in createdBy

Reply:
Gome form entity - changed to email in createdBy

Reply:
Field annotations added to createdBy